App/src/pages/dossier/DossierToReview.tsx

```diff
@@ -82,6 +82,16 @@ export default function DossiersToReview() {
                 >
                     My Dossiers
                 </Button>
+                <Button
```

Comment:
Nice job putting navigation buttons here, but can you also put the navigation buttons for the dossier browser to the my dossiers and dossier to review page (at the top of the page)?